sched/init/nx_start.c

```diff
@@ -610,6 +611,12 @@ void nx_start(void)
   }
 #endif
 
+#ifdef CONFIG_MM_IOB
+    /* Initialize IO buffering */
```

Comment:
There is an error in the indentatino here.  Should only be indented by two spaces.

Reply:
Let me provide a update.